media-gfx/openscad/openscad-2019.05.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="https://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
```

Comment:
Thanks for your changes. Please keep using the proper release tarball from `http://files.openscad.org/`

Reply:
@a17r I changed to github because of https, but just noticed files.openscad.org is also supporting https by now. I keep this SRC_URI then.